### PR TITLE
[POC] Installer helm3

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- implements helm3 changes

**Related issue(s)**
- see #8517
- Requires https://github.com/kyma-project/kyma/pull/8677 
- Requires https://github.com/kyma-project/kyma/pull/8678

### CODE REVIEW

File: installation/resources/installer-local.yaml [hunk 1]
```diff
@@ -194,19 +195,79 @@ spec:
...
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          HELM_2_BINARY=$(which helm)
+          HELM_3_BINARY=$(which helm3)
+          SECRET_NAME="helm-secret"
+          NAMESPACE="kyma-installer"
+          CHART_BLACKLIST=( "gateway-0.0.1" )
```
Comment (Szymongib) [inline installation/resources/installer-local.yaml:215]: Could you also add the `application-0.0.1` chart to the blacklist?
```suggestion
          CHART_BLACKLIST=( "gateway-0.0.1" "application-0.0.1" )
```

File: resources/monitoring/templates/prometheus/additionalPrometheusRules.yaml [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-{{ include "prometheus-operator.labels" $ | indent 4 -}}
-{{ if $config.additionalLabels }}
-{{ toYaml $config.additionalLabels | trim | indent 4 -}}
-{{- end }}
-spec:
-  groups:
-{{ toYaml $config.groups | indent 4 -}}
-{{- end }}
-{{- end }}
+  {{- range $name, $config := .Values.additionalPrometheusRules | fromYaml }}
```
Comment (rakesh-garimella) [inline resources/monitoring/templates/prometheus/additionalPrometheusRules.yaml:2]: There is discrepancy here from master. https://github.com/kyma-project/kyma/pull/8684 the PR is missing.

**Pull request merged**